### ISSUE TITLE
In the code, loss = outputs.loss throws an error stating that the loss attribute does not exist.

### ISSUE THREAD

**Issue by AIT55** [jeon185/LaViC#3]
I am using the llava-v1.6-mistral-7b-hf model and encountered an issue while reproducing the know_distillation.py script. I would appreciate your help in resolving it.

Take the training_step function as an example — the same issue also occurs in the validation_step. In the code, loss = outputs.loss throws an error stating that the loss attribute does not exist.

Upon investigation, I traced the root cause to the following line:

```
outputs = self.model.language_model(
    inputs_embeds=inputs_embeds,
    attention_mask=attention_mask,
    labels=labels
)
```

The outputs here come from self.model.language_model, which is defined as:

```
(language_model): MistralModel(
  (embed_tokens): Embedding(32007, 4096)
  (layers): ModuleList(
    (0-31): 32 x MistralDecoderLayer(
      ...
    )
  )
  ...
)
```

Since MistralModel itself does not compute the loss, it makes sense that outputs.loss does not exist.To address this, I modified the code as follows:

```
outputs = self.model(
    inputs_embeds=inputs_embeds,
    attention_mask=attention_mask,
    labels=labels
)

```
This change resolved the issue, and the code now runs as expected. However, I am not entirely sure whether this deviates from the original design intent of the script. I would greatly appreciate your insights on this matter and look forward to your response.
